saos-webapp/src/main/webapp/WEB-INF/view/analysis/analysis.jsp

```diff
@@ -1,6 +1,14 @@
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
 <%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
 
+<style>
+<!--
+    #ccCourtChart * .flot-x-axis .flot-tick-label {
+        font-size: 9px;
+    }
+-->
+</style>>
```

Comment:
if you don't use it, please consider removing it. Or at least correct closing tag `</style>>`